docs-chef-io/content/inspec/cloud_security.md

```diff
@@ -0,0 +1,212 @@
+
+The **Chef InSpec** reduces the burden of securing hybrid workloads by providing the ability to communicate directly with cloud-native APIs to audit your environment’s configuration. Since Chef InSpec already has facilities for scanning servers and virtual machines, it provides an effortless infrastructure to scan traditional and cloud-native environments all in one place. Using InSpec, you can gain visibility into the state of your apps, virtual machines, and configuration settings before you migrate your workloads into AWS. You can also configure new nodes and instances from a known state, define the steps required to configure an instance, and spin up new nodes against the desired state automatically, reducing the work needed and enforcing consistency of the newly launched instances.
+
+The **Chef Infra** allows you to transform infrastructure into code. Whether you are operating in the cloud, on-premises, or in a hybrid environment, Chef Infra automates how infrastructure is configured, deployed, and managed across your network, regardless of the size. It enables you to remediate the issues identified by Chef InSpec. In addition, it allows you to manage both traditional infrastructure and Virtual Machines (VMs) and the integrated cookbooks configured infrastructure and cloud resources.
+
+#### Approaches to the AWS Shared Responsibility Model
+
+Amazon’s responsibilities fall into two categories in the AWS shared responsibility model. The first is the hardware, including all Regions, Availability Zones, and Edge Locations of the AWS global infrastructure. The second category is managing the software for the compute, storage, database, and networking aspects of cloud computing.
+
+The customer, meanwhile, controls the data and configurations within that cloud environment. In the AWS shared responsibility model, customer responsibility falls into categories such as:
+
+- Customer data stored in the cloud
+- Platform, application, and identity & access management
+- Operating system, network, and firewall configuration
+- Data encryption, integrity, and authentication
+
+The **AWS Shared Responsibility Model** outlines how AWS commits to their cloud's "security" and where customers need to focus on security "in" the cloud.
+
+As a customer/company managing your own IT infrastructure (either bare metal or VMs) need to create a strategy and practice to manage all the elements depicted on the screen. AWS commits to the security of its platform and distributed infrastructure as a service to their customer. While benefiting from their expertise and scale, there is a massive boon for customers. It still means you need to have a plan to secure everything above the line.
```

Comment:
```suggestion
As a customer/company managing your own IT infrastructure (either bare metal or VMs), you need to create a strategy and practice to manage all the elements depicted on the screen. AWS commits to the security of its platform and distributed infrastructure as a service to their customer. While benefiting from their expertise and scale, there is a massive boon for customers. It still means you need to have a plan to secure everything above the line.
```

When it says "practice to manage all the elements depicted on the screen", what screen are we talking about? We should just list out all these elements.

Reply:
When it says "practice to manage all the elements depicted on the screen", what screen are we talking about? We should just list out all these elements. For accessibility reasons, we should depend on customers being able to see or read images.

The same thing applies for the phrase "secure everything above the line".